client/src/reader/containers/ReaderNotes/index.js

```diff
@@ -131,6 +131,15 @@ export class ReaderNotesContainer extends Component {
     );
   };
 
+  handleReadingGroupChange = rgId => {
```

Comment:
You could probably just use the `handleFilterChange` method for this to simplify things. Would just take a slight change to the shape of the data passed back from `GroupFilter`.